common/test/acceptance/tests/lms/test_bookmarks.py

```diff
@@ -66,35 +63,64 @@ def setUp(self):
             self.course_info['run']
         )
 
-        self.create_course_fixture()
+        self.courseware_page = CoursewarePage(self.browser, self.course_id)
+        self.bookmarks_page = BookmarksPage(self.browser, self.course_id)
+        self.course_nav = CourseNavPage(self.browser)
+
+    def _test_setup(self, num_chapters=2):
+        """
+        Setup test settings.
+
+        Arguments:
+            num_chapters: number of chapters to create in course
+        """
+        self.create_course_fixture(num_chapters)
 
         # Auto-auth register for the course.
         AutoAuthPage(self.browser, username=self.USERNAME, email=self.EMAIL, course_id=self.course_id).visit()
 
-        self.courseware_page = CoursewarePage(self.browser, self.course_id)
         self.courseware_page.visit()
-        self.bookmarks = BookmarksPage(self.browser, self.course_id)
 
-        # Use auto-auth to retrieve the session for a logged in user
-        self.session = requests.Session()
-        response = self.session.get(LMS_BASE_URL + "/auto_auth?username=STUDENT&email=student@example.com")
-        self.assertTrue(response.ok, "Failed to get session info")
+    def _bookmark_unit(self, index):
+        """
+        Bookmark a unit
+
+        Arguments:
+            index: unit index to bookmark
+        """
+        self.course_nav.go_to_section('TestSection{}'.format(index), 'TestSubsection{}'.format(index))
+        self.bookmarks_page.click_bookmark_unit_button()
+
+    def _bookmark_units(self, num_units):
+        """
+        Bookmark first `num_units` units by visiting them
 
-    def _bookmark_unit(self, course_id, usage_id):
-        """ Bookmark a single unit """
-        csrftoken = self.session.cookies['csrftoken']
-        headers = {'Content-type': 'application/json', "X-CSRFToken": csrftoken}
-        url = LMS_BASE_URL + "/api/bookmarks/v0/bookmarks/?course_id=" + course_id + '&fields=path'
-        data = json.dumps({'usage_id': usage_id})
+        Arguments:
+            num_units(int): Number of units to bookmarks
+        """
+        for index in range(num_units):
+            self._bookmark_unit(index)
+
+    def _breadcrumb(self, num_units):
+        """
+        Create breadcrumbs.
```

Comment:
I think this docstring should read similarly to the one above for `_bookmark_units`. This creates breadcrumbs for the first `num_units`.